pkg/scalers/metrics_api_scaler.go

```diff
@@ -35,7 +74,29 @@ func NewMetricsAPIScaler(resolvedEnv, metadata, authParams map[string]string) (S
 	if err != nil {
 		return nil, fmt.Errorf("error parsing metric API metadata: %s", err)
 	}
-	return &metricsAPIScaler{metadata: meta}, nil
+
+	if meta.enableTLS {
+		config, err := kedautil.NewTLSConfig(meta.cert, meta.key, meta.ca)
+		if err != nil {
+			return nil, err
+		}
+
+		transport := &http.Transport{TLSClientConfig: config}
+		return &metricsAPIScaler{
+			metadata: meta,
+			client: &http.Client{
+				Timeout:   3 * time.Second,
```

Comment:
```suggestion
				Timeout:   3 * time.Second,
```
How about creating `defaultTimeout` variable that will be used here and in L97?

Reply:
Whats L97? Sorry but I am not aware of this.

Reply:
No it is Line 89, when we return other Client:
```

  	return &metricsAPIScaler{
 		metadata: meta,
 		client: &http.Client{
 			Timeout: 3 * time.Second,
 		},
 	}, nil
```
I think it makes sense to have a default value for parameters that are used in two places

Reply:
oh got it :).